Engine/source/ts/tsMesh.cpp

```diff
@@ -238,7 +238,7 @@ void TSMesh::innerRender( TSMaterialList *materials, const TSRenderState &rdata,
       TORQUE_UNUSED(indexed);
       TORQUE_UNUSED(type);
 #endif
-
+*/
```

Comment:
Tossup there really. as-is, after the redefine of the TORQUE_UNUSED macro, all this does is lead to (void)sizeof(var)